src/server/jobs/mongodb_development_setup.md

```diff
@@ -0,0 +1,36 @@
+# Setting up MongoDB for development
```

Comment:
@chern This already exist in the top level readme. Shouldn't have it in two places. Else if we make a change in one this can get out of sync/date easily.